app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
Add type hint. 
HINT: use from __future__ import annotations

Reply:
please do as @ArsenPidhoretskyi said

here is an example for you
`alive: list[Animal] = [] `